src/openforms/variables/admin.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+
+from .models import ServiceFetchConfiguration
+
+
+@admin.register(ServiceFetchConfiguration)
+class ServiceFetchConfigurationAdmin(admin.ModelAdmin):
+    pass
```

Comment:
always give the admin some love in terms of `list_display`, `raw_id_fields` etc.